spec/controllers/queue_items_controller_spec.rb

```diff
@@ -5,8 +5,8 @@
     it "sets @queue_items" do
       current_user = Fabricate(:user)
       session[:user_id] = current_user.id
-      queue_item1 = QueueItem.create(user_id: current_user.id, video_id: 1, order: 1 )
-      queue_item2 = QueueItem.create(user_id: current_user.id, video_id: 2, order: 2)
+      queue_item1 = QueueItem.create(user_id: current_user.id, video_id: 1, position: 1 )
+      queue_item2 = QueueItem.create(user_id: current_user.id, video_id: 2, position: 2)
```

Comment:
If you are using Fabrication, you should stick to creating objects with object generator.